src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1384,7 +1388,7 @@ public void ReceiveFromV6BoundToSpecificV4_NotReceived()
         }
 
         [Fact]
-        [PlatformSpecific(~PlatformID.Linux)]
+        [PlatformSpecific(~(PlatformID.Linux | PlatformID.OSX))]
```

Comment:
Should this be PlatformSpecific(PlatformID.Windows) now?  Or do we believe that other Unix-based platforms will be able to support this, just not Linux and OS X?